src/lib/Form/Type/DateTimeFormat/DateTimeFormatType.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Type\DateTimeFormat;
+
+use EzSystems\EzPlatformAdminUi\Form\DataTransformer\DateTimeFormatTransformer;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Form Type allowing to select DateTime Format.
+ */
+class DateTimeFormatType extends AbstractType
+{
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     * @param array $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('date_format', ChoiceType::class, [
+                'label' => $options['date_format_label'],
+                'choices' => $options['date_format_choices'],
```

Comment:
I'm not convinced with this. Use `Symfony\Component\Form\ChoiceList\Loader\ChoiceLoaderInterface` to generate list of choices. Especially in this case where it's served from `DateTimeFormatRepository`.